test/jdk/sun/tools/jhsdb/JShellHeapDumpTest.java

```diff
@@ -158,6 +158,8 @@ public static void launchJshell() throws IOException {
                 launcher.addVMArgs(Utils.getFilteredTestJavaOpts("-XX:\\+UseSerialGC"));
             }
             ProcessBuilder pb = new ProcessBuilder(launcher.getCommand());
+            pb.command().add("-J-Duser.language=en");
+            pb.command().add("-J-Duser.country=US");
```

Comment:
How about a comment:

/* Needed so we can properly parse the "Welcome to JShell" output. */

Reply:
Thank you for your comment.
I added a comment to the code.

Thanks,
Kimura Yukihiro